site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Build the Android App"
+date: 2018-05-20T11:35:32-04:00
+weight: 4
+subsection: Build Your Own App
+---
+
+Android requires that all apps be digitally signed with a certificate before they can be installed, so to distribute your Android application via the Google Play Store, you'll need to generate a signed release APK.
+
+#### Generating a Signing Key
+
+To generate the signed key, we'll be using **keytool** which comes with the JDK required to develop for Android.
+
+```sh
+$ keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+```
+
+The above command prompts you for passwords for the keystore and key (make sure you use the same password for both), and asks you to provide the Distinguished Name fields for your key. It then generates the keystore as a file called my-release-key.keystore.
+
+The keystore contains a single key, valid for 10000 days. The alias is a name that you will use later when signing your app, so remember to take a note of the alias.
+
+**Remember to keep your keystore file private and never commit it to version control.**
+
+#### Setting up Gradle Variables
+
+- Place the *my-release-key.keystore* file under a directory that you can access. It can be in your home directory or even under *android/app* in the project folder so long as it is not checked in.
+- Edit the file \~/.gradle/gradle.properties, or create it if one does not exist, and add the following:
+
+    ```sh
+    MATTERMOST_RELEASE_STORE_FILE=/full/path/to/directory/containing/my-release-key.keystore
+    MATTERMOST_RELEASE_KEY_ALIAS=my-key-alias
+    MATTERMOST_RELEASE_PASSWORD=*****
+    ```
+
+**Replace `/full/path/to/directory/containing/my-release-key.keystore` with the full path to the actual keystore file and `\********` with the actual keystore password.**
+
+**Once you publish the app on the Play Store, you will need to republish your app under a different package id (losing all downloads and ratings) if you change the signing key at any point, so backup your keystore and don't forget the password.**
```

Comment:
Make "backup your keystore and don't forget the password" into a separate sentence.